<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutDialog</class>
 <widget class="QDialog" name="ShortcutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Shortcut</string>
  </property>
  <property name="whatsThis">
   <string>Press your shortcut keys and select OK</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="shortcutsGroup">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="primaryClearButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Press to clear the key sequence buffer.</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="keyLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="6"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShortcutDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
